packages/block-library/src/list-item/edit.js

```diff
@@ -30,7 +30,7 @@ import {
 } from './hooks';
 import { convertToListItems } from './utils';
 
-function IndentUI( { clientId } ) {
+export function IndentUI( { clientId } ) {
```

Comment:
This export is needed to share it with the mobile variant of the `ListItem` block